modules/stuffed/tomcat-resource.json

```diff
@@ -73,9 +73,13 @@
     {"name":"org.apache.tomcat.websocket.server.LocalStrings"}
   ],
   "resources":[
-    {"pattern":".*/mbeans-descriptors.xml$"},
-    {"pattern":".*/*.properties$"},
-    {"pattern":".*/*.dtd$"},
-    {"pattern":".*/*.xsd$"}
+    {"pattern":"^org/apache/tomcat/.*mbeans-descriptors\\.xml$"},
+    {"pattern":"^org/apache/catalina/.*mbeans-descriptors\\.xml$"},
+    {"pattern":"^org/apache/coyote/.*mbeans-descriptors\\.xml$"},
+    {"pattern":"^org/apache/catalina/.*\\.properties$"},
+    {"pattern":"^jakarta/servlet/resources/.*"},
```

Comment:
So we really need to include all of the XSDs and the DTDs?

Reply:
I am not sure, if for typical use case we could keep only the latest version for example, would be nice, but I would need Tomcat team guidelines for that cc @markt-asf.

Reply:
They are only required if xmlValidation (or tldValidation) is enabled. I think you'll get warnings if they are missing.

Reply:
Interesting, let me have another look to see if this should be shipped by default or not.

Reply:
After a deeper look, I have chosen to remove `jakarta/servlet/resources/*` resources from the default configuration of `tomcat-embed-core`, since those optional resources weight `1.5M`! Users leveraging them will just have to configure something like `{"pattern":"^jakarta/servlet/resources/.*"}`, not that different from what people need to do for JNI or extra reflection configuration.